docs/learning/howto/acls/group-jobname.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Group/Project/JobName Execute Only"
+order: 500
+---
+
+# Group/Project/JobName Execute Only
```

Comment:
We should also allow read access to the project Key Storage, or note that if the jobs uses Key Storage entries it will need to use something to provide those rights.